usaspending_api/spending_explorer/v2/filters/type_filter.py

```diff
@@ -158,7 +149,7 @@ def type_filter(_type, filters, limit=None):
     # Apply filters to queryset results
     alt_set, queryset = spending_filter(alt_set, queryset, filters, _type)
 
-    if _type == "recipient" or _type == "award" or _type == "award_category" or _type == "agency_type":
+    if _type == "recipient" or _type == "award" or _type == "award_category":
```

Comment:
I understand you only removed the `agency_type` value, but is there any reason to not change this to a set instead of multiple `or` statements while we're at it? This change could allow us to short-circuit this check as soon as a match is found instead of needing to run every check with the `or` statements.

```python
if _type in {"recipient", "award", "award_category"}:
```